installers/container-sirepo-base/radiasoft-download.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+container_sirepo_base_main() {
+    declare user=$1
+    case $user in
+        "root")
```

Comment:
Quotes are important in bash. These are unnecessary, and even if they were necessary, they should have been single quotes.

Reply:
26febff